App/config/packages/translation.yaml

```diff
@@ -0,0 +1,6 @@
+framework:
+    default_locale: 'en'
```

Comment:
I though the default locale would be 'fr' as Arthur ask to starts with French version of the application 👍